### PR TITLE
Add HEAT-Seq workflow

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -12,3 +12,6 @@ recursive-include resolwe_bio/tests *.py
 recursive-include resolwe_bio/kb/tests *.py
 recursive-include resolwe_bio/tests/files *
 prune resolwe_bio/tests/files/large
+
+# added by check_manifest.py
+recursive-include resolwe_bio *.yml
```
Comment (kostko) [inline MANIFEST.in]: Remove this and update `setup.py`.

File: resolwe_bio/processes/workflows/vc/heat_seq.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        run: alignment-bwa-mem
+        input:
+          genome: '{{input.genome}}'
+          reads: '{{steps.hsqutils_trimming}}'
+          m: true
+      - id: hsqutils_dedup
+        run: hsqutils-dedup
+        input:
+          alignment: '{{steps.align}}'
+          reads: '{{_steps.trimmomatic_filtering}}'
```
Comment (kostko) [inline resolwe_bio/processes/workflows/vc/heat_seq.yml]: Change this to `steps`.

**Comment by kostko** [comment]
I've pushed workflow improvements to Resolwe master.

File: MANIFEST.in [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-# include tests and files needed by tests (except large files)
-include tox.ini
-include .pylintrc
-include tests/.env
-recursive-include tests *.py *.yml *.yaml *.rst
-recursive-include resolwe_bio/tests *.py
-recursive-include resolwe_bio/kb/tests *.py
-recursive-include resolwe_bio/tests/files *
-prune resolwe_bio/tests/files/large
+ # include meta data
```
Comment (mstajdohar) [inline MANIFEST.in]: Why did you add a space in front of all the lines?

File: resolwe_bio/processes/workflows/heat_seq.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
+- slug: workflow-heat-seq
+  category: workflow:primary
+  name: HEAT-seq workflow
```
Comment (mstajdohar) [inline resolwe_bio/processes/workflows/heat_seq.yml]: Maybe change to "HEAT-seq Pipeline".

File: resolwe_bio/tests/processes/test_variant_calling.py [hunk 5]
```diff
@@ -1,5 +1,6 @@
 # pylint: disable=missing-docstring
```
Comment (mstajdohar) [inline resolwe_bio/tests/processes/test_variant_calling.py]: Maybe we should move this from `.../tests/processes/...` to `.../tests/workflows/...`.

File: MANIFEST.in (hunk 3 above)
Comment (jkokosar) [inline MANIFEST.in]: That was a mistake when copy-paste-ing.

File: resolwe_bio/tests/processes/test_variant_calling.py (hunk 5 above)
Comment (jkokosar) [inline resolwe_bio/tests/processes/test_variant_calling.py]: Changed as suggested.

**Comment by mstajdohar** [comment]
Fix tests https://travis-ci.org/genialis/resolwe-bio/builds/181956412

**Comment by jkokosar** [comment]
@mstajdohar test passes on local machine. Please test if it works for you: `./manage.py test resolwe_bio.tests.workflows`
